examples/collections/mainwindow.cpp

```diff
@@ -262,6 +263,9 @@ MainWindow::MainWindow(QWidget *parent)
     dlg->addItem(new DFeatureItem(QIcon::fromTheme("dialog-warning"), "对话框", "普通对话框(DDialog)，用于需要用户处理事务，又不希望跳转页面以致打断工作流程时。", dlg));
     dlg->addItem(new DFeatureItem(QIcon::fromTheme("dialog-warning"), "DSpinner", "所有需要用户等待的地方，且没有具体的等待时间，不知道进度，可能很快也可能需要比较久。", dlg));
     dlg->addItem(new DFeatureItem(QIcon::fromTheme("dialog-warning"), "进度条", "进度条(DWaterProgress)一种带趣味的展示形式，作用是减少用户枯燥的等待。", dlg));
+    
+    qApp->licenseDialog()->setJsonFile(":/resources/data/example-license.json");
+    qApp->licenseDialog()->setLicenseContentPath(":/resources/data");
```

Comment:
1. setJsonFile/setLicenseContentPath 函数名还是再想想
2. 可以直接作为 DApplication 的接口，类似 setProductName/setApplicationDescription 接口一样调用


```suggestion
    qApp->setApplicationCreditsInfo(":/resources/data/example-license.json");
    qApp->setLicenseSearchPath(":/resources/data");
```

Reply:
void setApplicationCreditsFile(const QString &file);
    void setApplicationCreditsContent(const QByteArray &content);
    void setLicensePath(const QString &path);

Reply:
a->setApplicationLicense(QObject::tr("2023 UnionTech Software Technology Co., Ltd."));
    a->setApplicationCreditsFile(":/resources/data/example-license.json");
    a->setLicensePath(":/resources/data");